### DISCUSSION TITLE
Update on import creates a new object

### DISCUSSION THREAD

**Discussion by SimonSkotheimsvik** [Micke-K/IntuneManagement#89] [Q&A]
I am looking into renaming some Intune objects. I have exported the objects and made a change in the displayName before doing an import. I am selecting the import type of Update, but still the imported object is created as a new update instead of renaming the existing one. By doing a compare of the Intune Object and the JSON Backup Object I can confirm the same Id - only the displayName has changed between Intune and the JSON file. I notice README.md has a note on a setting named "Allow update on import" in settings which I can't find. Should it be possible to do a rename of an Intune object this way?

**Comment by Micke-K** [comment]
Hello,

This is not how it is designed. It will only look for a policy with the same name and not based on the id. 

A solution for this would obviously require that the policy is imported in the same environment as it was exported but would also require a redesign of the import scenario.

Cheers!

**Comment by Micke-K** [comment]
Hello!

A few questions regarding this so I understand what you are trying to do here.

Where are you doing this? Same tenant as you exported the policies?

Why are you renaming them? How many?

I'm trying to figure out if there is a need for Bulk Rename or some other kind of rename functionality or if this is just a one time thing. 

Cheers!

**Comment by SimonSkotheimsvik** [comment]
The intention was to do it on the same tenant as the export in order to have a new and consistent naming standard across the different policies in the  tenant.
